game/templates/game/lesson_detail.html

```
@@ -0,0 +1,447 @@
+    {% if board_examples %}
+
+    <div class="board-demo">
+
+        <h2>♟ Interactive Board Example</h2>
+
+        <h3 id="example-title"></h3>
+
+        <div id="demo-board"></div>
+
+        <button
+            id="next-example-btn"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)